src/pages/home/report/ReportActionItemMessageEdit.tsx

```diff
@@ -80,23 +72,34 @@ const messageEditInput = 'messageEditInput';
 
 const isMobileSafari = Browser.isMobileSafari();
 
-function ReportActionItemMessageEdit(props) {
+function ReportActionItemMessageEdit({
+    action,
+    draftMessage,
+    reportID,
+    index,
+    shouldDisableEmojiPicker = false,
+    preferredSkinTone = CONST.EMOJI_DEFAULT_SKIN_TONE,
+    forwardedRef = () => {},
+}: ReportActionItemMessageEditProps & {
+    forwardedRef: ForwardedRef<TextInput & HTMLTextAreaElement>;
```

Comment:
Instead of forwardedRef, use ref as a second argument, this aligns with the guidelines better